airbyte-integrations/connectors/source-mysql/src/main/java/io/airbyte/integrations/source/mysql/MySqlSource.java

```diff
@@ -211,7 +193,7 @@ public List<AutoCloseableIterator<AirbyteMessage>> getIncrementalIterators(final
           fetcher,
```

Comment:
I am getting compilation errors here

Reply:
Again. I don't see an error.
Can you check you got the change in `MySqlCdcProperties`?
```static Properties getDebeziumProperties(final JdbcDatabase database)```

Reply:
all good. User error.